roles/postgres/tasks/main.yml

```diff
@@ -56,3 +56,29 @@
   with_items: "{{ postgres_users }}"
   become: true
   become_user: postgres
+
+- name: Add a service template that allows creating backups of postgresql databases
+  ansible.builtin.copy:
+    src: postgres-backups@.service
+    dest: /usr/lib/systemd/system/postgres-backups@.service
+    mode: u=rw,g=r,o=r
+  become: true
+
+- name: Add timers triggering backups of postgresql databases
+  ansible.builtin.template:
+    src: postgres-backups@.timer.j2
+    dest: /usr/lib/systemd/system/postgres-backups@{{ item.database }}.timer
+  with_items: "{{ postgres_users }}"
+  when:
+    - item.backup_schedule is defined and item.backup_schedule
```

Comment:
nit: there's no need for a list if it's a single line condition

Reply:
I did not know that! Thanks!